store/index.js

```diff
@@ -286,9 +286,7 @@ export const actions = {
   async detectHalfWrapped({ state, commit }, { ketherContract, nftContract, numBlocks }) {
     // FIXME: Probably should move this into Wrap or a ResumeWrap component, no need to pollute global state
     const account = state.activeAccount;
-    console.log("statrt")
     if (!account) {
-      console.error("Can't detect half-wrapped ads without an active account. Connect a wallet.");
```

Comment:
we now call this on every page load so might as well not pollute console with errors.